libs/core/src/lib/avatar/avatar.component.ts

```diff
@@ -200,7 +203,11 @@ export class AvatarComponent implements OnChanges, OnInit, CssClassBuilder {
     }
 
     /** @hidden */
-    constructor(private _elementRef: ElementRef, private _cdr: ChangeDetectorRef) {}
+    constructor(
+        private _elementRef: ElementRef,
+        private _cdr: ChangeDetectorRef,
+        private readonly _skeletonConsumer: SkeletonConsumer
```

Comment:
just thinking: it's quite not obvious from the first glance, why this service is imported without being references even once. Maybe in order to be more explicit it would be better to pass the elementRef in the constructor of the component instead of using DI? That will visually tie the component with the logic of the consumer

Reply:
I do not know, whole idea behind the Consumers was that it can be low config implicit. We could add config function to the consumers and that could be optional thing